src/Endpoints/class-analytics-posts-api-proxy.php

```diff
@@ -41,16 +41,19 @@ public function get_items( WP_REST_Request $request ): stdClass {
 	 * @return array<stdClass> The generated data.
 	 */
 	protected function generate_data( array $response ): array {
-		$date_format = get_option( 'date_format' );
-		$result      = array_map(
-			static function( stdClass $item ) use ( $date_format ) {
+		$date_format    = get_option( 'date_format' );
+		$stats_base_url = trailingslashit( 'https://dash.parsely.com/' . esc_js( $this->parsely->get_api_key() ) ) . 'find';
```

Comment:
no real need to escape this here since it is not because used in output

```suggestion
		$stats_base_url = trailingslashit( 'https://dash.parsely.com/' . $this->parsely->get_api_key() ) . 'find';
```